skbeam/core/accumulators/binned_statistic.py

```diff
@@ -70,8 +70,13 @@ def __init__(self, sample, statistic='mean',
             A sequence of lower and upper bin edges to be used if the
             edges are not given explicitely in `bins`. Defaults to the
             minimum and maximum values along each dimension.
-
+        mask : array_like
+            array of ones and zeros with the same shape as `sample`.
+            Values with mask==0 will be ignored.
         """
+        if mask is None:
+            mask = np.ones_like(sample)
```

Comment:
I also think that mask is data dependent. I would vote for keeping the mask at a higher level (i.e. binnedstatistic2D)